apps/condo/domains/billing/schema/BillingReceiptFile.test.js

```diff
@@ -0,0 +1,254 @@
+                        const [file] = await createTestBillingReceiptFile(admin, receiptByService, context)
+                        await expectToThrowAccessDeniedErrorToObj(async () => {
+                            await updateTestBillingReceiptFile(user, file.id)
+                        })
+                    })
+                })
+            })
+
+            test('anonymous can\'t', async () => {
+                const [file] = await createTestBillingReceiptFile(admin, receiptByAdmin, context)
+
+                await expectToThrowAuthenticationErrorToObj(async () => {
+                    await updateTestBillingReceiptFile(anonymous, file.id)
+                })
+            })
+        })
+
+        describe('hard delete', () => {
+            let file
+            beforeAll(async () => {
+                [file] = await createTestBillingReceiptFile(admin, receiptByAdmin, context)
+            })
+            test('admin can\'t', async () => {
+                await expectToThrowAccessDeniedErrorToObj(async () => {
+                    await BillingReceiptFile.delete(admin, file.id)
+                })
+            })
+
+            test('user can\'t', async () => {
+                await expectToThrowAccessDeniedErrorToObj(async () => {
+                    await BillingReceiptFile.delete(user, file.id)
+                })
+            })
+
+            test('anonymous can\'t', async () => {
+                await expectToThrowAccessDeniedErrorToObj(async () => {
+                    await BillingReceiptFile.delete(anonymous, file.id)
+                })
+            })
+        })
+
+        describe('read', () => {
+            let file
+            beforeAll(async () => {
+                [file] = await createTestBillingReceiptFile(admin, receiptByAdmin, context)
+            })
+            test('admin can', async () => {
+                const objs = await BillingReceiptFile.getAll(admin, {}, { sortBy: ['updatedAt_DESC'] })
+
+                expect(objs.length).toBeGreaterThanOrEqual(1)
+                expect(objs).toEqual(expect.arrayContaining([
+                    expect.objectContaining({
+                        id: file.id,
+                        context: file.context,
+                        receipt: file.receipt,
+                        controlSum: file.controlSum,
+                    }),
+                ]))
+            })
+
+            test('service can', async () => {
+                const objs = await BillingReceiptFile.getAll(integrationUser, {}, { sortBy: ['updatedAt_DESC'] })
+
+                expect(objs.length).toBeGreaterThanOrEqual(1)
+                expect(objs[0]).toMatchObject({
+                    id: file.id,
+                    context: file.context,
+                    receipt: file.receipt,
+                    controlSum: file.controlSum,
+                })
+            })
+
+            test('anonymous can\'t', async () => {
```

Comment:
You need to add 2 cases employee with the same organization - can read
and employee from another organization - can't